tools/minicluster/print_utils.py

```diff
@@ -10,16 +10,16 @@ class bcolors:
 
 
 def okblue(message):
-    print bcolors.OKBLUE + message + bcolors.ENDC
+    print(bcolors.OKBLUE + message + bcolors.ENDC)
```

Comment:
This is not a syntax error in Python 2, but Python 3 doesn't allow it. Can you update your commit message/description to say that this change is needed for Python 3 compatibility?